frontend/src/metabase/visualizations/lib/action.unit.spec.ts

```diff
@@ -16,6 +20,7 @@ describe("performAction", () => {
     const extraProps = {
       dispatch: jest.fn(),
       onChangeCardAndRun: jest.fn(),
+      onUpdateQuestion: jest.fn(),
```

Comment:
This is just to fix type check errors because of TypeScript type inference for `.js` files - it assume that `onUpdateQuestion` is required, when it really isn't. We could've also refactored `action.js` to TypeScript here.